vm/src/obj/objbytes.rs

```diff
@@ -201,3 +203,58 @@ fn bytes_iter(obj: PyBytesRef, _vm: &VirtualMachine) -> PyIteratorValue {
         iterated_obj: obj.into_object(),
     }
 }
+
+fn compare_slice(a: &[u8], b: &[u8]) -> bool {
+    for (i, j) in a.iter().zip(b.iter()) {
+        if i != j {
+            return false;
+        }
+    }
+    return true;
+}
+
+fn compare_vec(a: &Vec<u8>, b: &Vec<u8>) -> bool {
+    let a_len = a.len();
+    let b_len = b.len();
+    for (n, i) in a.iter().enumerate() {
+        if n + b_len <= a_len && *i == b[0] {
+            if compare_slice(&a[n..n + b_len], b) {
+                return true;
+            }
+        }
+    }
+    false
+}
```

Comment:
for compare_vec and compare_slice i didn't find an easier way to do it. maybe it exists ?

Reply:
If compare_slice is just checking that all the elements in each are equal, I think you could just use `slice_a == slice_b`